keg_elements/templates/keg_elements/forms/horizontal.html

```diff
@@ -38,45 +38,26 @@
 {%- endmacro %}
 
 {% macro description(field) -%}
-  <div class="description">
-        <a type="button" class="btn btn-default" rel="kegel-field-description"
-            title="{{ field.label.text }} Field" tabindex="-1"
-            data-field-description-content="#{{ field.id }}_description_content">?</a>
-    <div id="{{ field.id }}_description_content" class="hide">
+    <small class="form-text text-muted description">
```

Comment:
Did you verify that these classes are valid for Bootstrap 3? I think the docs said help-block was the Bootstrap 3 class. Also, we should probably enable using Bootstrap 4 for keg-auth for consistency.

Reply:
I verified locally but didn't think to check my bootstrap version. Looks like `help-block` should be used instead of `form-text` according to: https://getbootstrap.com/docs/4.0/migration/#forms-1. I'll get on that. `text-muted` is bootstrap 3 compatible: https://getbootstrap.com/docs/3.3/css/#helper-classes-colors.